README.md

```
@@ -52,33 +57,29 @@ $ printf "%s" "$CANVAS_API_TOKEN" | ccc course setup --no-interactive \
 ✓ Course ID 12345 validated
-✓ Course configuration saved as block: ccc-course-cs101
+✓ Course configuration saved as block: ccc-course-12345-cs101
 ```
 
 ### 2. Run a correction manually
 
 Use `ccc course run` for batch or single-submission runs.
 
 ```bash
-$ ccc course run 98765 --course ccc-course-cs101
+$ ccc course run 98765 --course ccc-course-12345-cs101
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)